nifi-frontend/src/main/frontend/apps/update-attribute/src/app/pages/update-attribute/ui/rule-listing/rule-listing.component.scss

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+:host {
+    width: 100%;
+}
+
+.rule-listing {
+    .cdk-drag-disabled {
+        font-style: normal;
```

Comment:
Is this needed? What is the purpose?

Reply:
Yes it is because there is an existing `font-style` applied to `cdk-drag-disabled` globally that results in affecting the edit rule for inside the expansion panel. The other instances of `cdk-drag-disable` is a simple listing and not applied to an expansion panel.

Reply:
It has been removed int the latest. Thanks!